### ISSUE TITLE
nano node .deb package dependency mismatch on debian testing (bullseye)

### ISSUE THREAD

**Issue by oiime** [nanocurrency/nano-node#3581]
### Summary

Hello, `qt5-default` no longer comes with debian 11, that causes installation difficultly as it either needs to be installed manually or using --force-all on the package which can cause issues later.

### Node version

22.1

### Build details

dpkg --force-all -i nano-node-V22.1-Linux.deb

from log
[2021-Dec-04 09:17:09.713984]: Node starting, version: V22.1
[2021-Dec-04 09:17:09.714030]: Build information: d91016b "GNU C++ version " "7.5.0" "BOOST 107000" BUILT "Jun 11 2021"
[2021-Dec-04 09:17:09.714046]: Database backend: LMDB 0.9.25

### OS and version

5.10.0-9-amd64 #1 SMP Debian 5.10.70-1 (2021-09-30) x86_64 GNU/Linux

### Steps to reproduce the behavior

1. install debian 11
2. download nano-node-V23.0RC1-Linux.deb
3. dpkg -i nano-node-V23.0RC1-Linux.deb

### Expected behavior

debian package managing to install on what would eventually be debian stable

### Actual behavior

```
dpkg: dependency problems prevent configuration of nano-node:
 nano-node depends on qt5-default; however:
  Package qt5-default is not installed.
```

### Possible solution

1. have a version specific .deb with different dependencies 
2. break installation into several packages as qt is not needed for just running a node (as far as i understood from the code)
